scene/debugger/scene_debugger.h

```diff
@@ -31,17 +31,13 @@
 #ifndef SCENE_DEBUGGER_H
 #define SCENE_DEBUGGER_H
 
-#include "core/input/shortcut.h"
```

Comment:
This is still needed

Reply:
Why though? The class is not even used in this file.

Reply:
Well for some reason it's not needed anywhere else. ViewPanner is included in cpp files, with not `shortcut.h` include, e.g. in `polygon_2d_editor_plugin.cpp`. It has problems only here.